velox/common/memory/Memory.cpp

```diff
@@ -271,8 +273,7 @@ std::shared_ptr<MemoryPool> MemoryManager::addRootPool(
       options);
   pools_.emplace(poolName, pool);
   VELOX_CHECK_EQ(pool->capacity(), 0);
-  arbitrator_->growCapacity(
-      pool.get(), std::min<uint64_t>(poolInitCapacity_, maxCapacity));
+  arbitrator_->addPool(pool);
```

Comment:
Should we add a test to ensure the pools are identical in `MemoryManager::pools_` and `SharedArbitrator:candidates_`?